### ISSUE TITLE
[campfire] commands to build and launch the web client

### ISSUE THREAD

**Issue by ten3roberts** [AmbientRun/Ambient#570]
Add commands to campfire to build, serve and launch the web client in its entirely by running all the necessary commands in the correct directory

```
cargo cf example serve [...]
cargo cf web serve-dev
cargo cf web launch-chrome [ip]
```

#151

**Comment by philpax** [comment]
Can we close this issue now that #595 has landed?

**Comment by ten3roberts** [comment]
No. I only had time to add `build` and `open-browser` as I had to move on the CI deploy.

The `serve` is still missing, as well as `open-browser` on linux and windows

**Comment by philpax** [comment]
Just to confirm, it's just windows serve missing now, right?

**Comment by ten3roberts** [comment]
Yes
